apps/website/src/components/events/fall-carnival-2023/IntroSection.tsx

```diff
@@ -0,0 +1,79 @@
+import NextLink from "next/link";
+import Image from "next/image";
+
+import teaserImage from "@/assets/events/fall-carnival-2023/teaser2.png";
+
+import Section from "@/components/content/Section";
+import Heading from "@/components/content/Heading";
+import Link from "@/components/content/Link";
+
+type IntroSectionProps = {
+  showTicket?: boolean;
+  showLinkToClaimTicket?: boolean;
+};
+
+export function IntroSection({
+  showTicket = false,
+  showLinkToClaimTicket = false,
+}: IntroSectionProps) {
+  return (
+    <Section
+      dark
+      className="bg-[#4E1362]"
+      containerClassName={`flex flex-col md:flex-row-reverse gap-12 lg:gap-24 items-center justify-center ${
+        showTicket ? "" : "text-center"
+      }`}
+    >
+      {showTicket && (
+        <div>
+          <NextLink
+            className="hover:scale-102"
+            href="/events/fall-carnival-2023"
+          >
+            <Image
+              src={teaserImage}
+              alt=""
+              width={400}
+              className="max-w-[400px]"
+            />
+          </NextLink>
+        </div>
+      )}
+
+      <div>
+        <div className="mb-2 inline-block rounded-lg bg-white/90 px-2 py-0.5 text-xs uppercase text-[#4E1362]">
+          Upcoming event
+        </div>
+
+        <Heading
+          style={{
+            // NOTE: react ts typings don't know about text-wrap yet so we have to disable the rule
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            textWrap: "balance",
```

Comment:
Do we want to add this as a default to headings or at least configure it to be used in tailwind? Browser support is still pretty sparse, but its such a nice feature https://caniuse.com/css-text-wrap-balance